configuration/multimedia.md

```diff
@@ -386,3 +384,109 @@ listenAndAnswer(null, null)
 listenAndAnswer("javasound", "sonos:PLAY5:kitchen")
 listenAndAnswer("voskstt", "voicerss", "voicerss:frFR_Axel", "system,rulehli", "javasound", "sonos:PLAY5:kitchen", "fr-FR", "listeningItem")
 ```
+
+### The Built-in Interpreter
+
+These interpreter is available by default and includes a build-it grammar for English, German, French and Spanish that you can extend using the voiceSystem metadata. These are some examples of the built-in English grammar:
+
+```text
+increase the <item name>
+decrease the <item name>
+set the color of the <item name> to red
+put the <item name> to next
+put the <item name> to previous
+play the <item name>
+pause the <item name>
+rewind the <item name>
+fast forward the <item name>
+start the <item name>
+stop the <item name>
+refresh the <item name>
+```
+
+For exact built-in grammar you can refer to the [source code](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/text/StandardInterpreter.java#L42).
```

Comment:
At least the line 42 is probably not correct. Did you want to point to L92, the beginning of the createRules method?